### PR TITLE
feat (schematics): allow comments in angular.json file (Issue #480)

### PR DESCRIPTION
**Pull request by atao60**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x ] The commit message follows our guidelines: https://github.com/scullyio/scully/blob/master/CONTRIBUTING.md#commit
- [ x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: #480

## What is the new behavior?

Allow comments in angular.json file with Scully schematics
In fact allow json5 syntax as Angular itself.
Note. ATM json5 is allowed only with angular.json files. But it'd be very easy to extend it to any json file of your choice.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x ] No

## Other information
This is a sequel of pr #531, as in the meantime the folder schematics/scully has been moved to libs/scully-schematics.

### CODE REVIEW

**Comment by atao60** [comment]
The command `npx nx run-many --target=build --all --prod` is working on my workstation but failed during the CI / build on github as it can't find dependency for @types/js-yaml
Any clue?

**Comment by atao60** [comment]
It seems this pr is stuck. May be because replacing pr #531 just after @SanderElias assigned the review of this last one to @jorgeucano.
@SanderElias, may I ask the review of the present one to be reassigned to @jorgeucano?

File: libs/scully-schematics/package.json [hunk 1]
```diff
@@ -15,7 +15,9 @@
     "test": "tsc -p tsconfig.spec.json && jasmine src/**/*_spec.js",
     "publish:patch": "tsc -p ./tsconfig.json && npm version patch && npm publish --access public",
     "publish:minor": "tsc -p ./tsconfig.json && npm version minor && npm publish --access public",
-    "publish:major": "tsc -p ./tsconfig.json && npm version major && npm publish --access public"
+    "publish:major": "tsc -p ./tsconfig.json && npm version major && npm publish --access public",
+    "clean": "rimraf spec/**/*.js spec/**/*.d.ts spec/**/*.js.map src/**/*.js src/**/*.d.ts src/**/*.js.map",
```
Comment (jorgeucano) [inline libs/scully-schematics/package.json:19]: Why you add this?

Comment (atao60) [reply libs/scully-schematics/package.json:19]: It's just a dev convenience to get a clean state on workstation. Not needed during CI Build.
Can be removed if you wish.

Comment (jorgeucano) [reply libs/scully-schematics/package.json:19]: please remove this and I will approve! thanks!

Comment (atao60) [reply libs/scully-schematics/package.json]: @jorgeucano Done

**Pull request merged**